merkletree/pad.go

```diff
@@ -114,8 +114,9 @@ func (pad *PAD) GetSTR(epoch uint64) *SignedTreeRoot {
 func (pad *PAD) TB(key string, value []byte) (*TemporaryBinding, error) {
 	//FIXME: compute private index twice
 	//it would be refactored after merging VRF integration branch
-	index, _ := pad.computePrivateIndex(key)
-	tb := pad.currentSTR.sig
+	cStr := pad.currentSTR
```

Comment:
> What do you think about lastSTR or latestSTR?

`latestSTR` sounds ok.  While we're at it, maybe do `s/generateNextSTR/SignTreeRoot/` as well.